src/Symfony/Component/Translation/Formatter/DefaultMessageFormatter.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Translation\Formatter;
+
+use Symfony\Component\Translation\MessageSelector;
+
+/**
+ * @author Guilherme Blanco <guilhermeblanco@hotmail.com>
+ */
+class DefaultMessageFormatter implements MessageFormatterInterface
+{
+    /**
+     * @var MessageSelector
+     */
+    protected $selector;
+
+    /**
+     * @param MessageSelector $selector Message selector to choose pluralization messages.
+     */
+    public function __construct(MessageSelector $selector = null)
+    {
+        $this->selector = $selector ?: new MessageSelector();
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function format($locale, $id, $number = null, array $arguments = array())
+    {
+        $pattern = ($number !== null)
```

Comment:
should be on one line, without the parenthesis around the condition and null should be on the left.